tests/all-packages-published.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# Copyright (C) 2013 Valve Corporation
+# Copyright (C) 2018 Collabora Ltd.
+#
+# SPDX-License-Identifier: MIT
+# (see COPYING)
+
+from __future__ import print_function
+
+import sys
+
+if __name__ == '__main__':
+	source_pkgs = set()
```

Comment:
tabs in a python file, this is not PEP-8 compliant :)

Reply:
The coding style is the same as `build-runtime.py`. I'd be happy to make it more standard.